transforms/code/repo_level_ordering/ray/Makefile

```diff
@@ -0,0 +1,51 @@
+# Define the root of the local git clone for the common rules to be able 
+# know where they are running from.
+REPOROOT=../../../..
+# Include a library of common .transform.* targets which most
+# transforms should be able to reuse.  However, feel free
+# to override/redefine the rules below. 
+
+include $(REPOROOT)/transforms/.make.transforms
+
+TRANSFORM_NAME=repo_level_order
+DOCKER_IMAGE_VERSION=${REPO_LVL_ORDER_RAY_VERSION}
+
+venv::	.transforms.ray-venv
+
+test::	.transforms.ray-test
+
+clean:: .transforms.clean
+
+image:: .transforms.ray-image
+
+test-src:: .transforms.test-src
+
+setup:: .transforms.setup
+
+test-image:: .transforms.ray-test-image
+
+build:: build-dist image
+
+publish:: publish-dist publish-image
+
+publish-image:: .transforms.publish-image-ray
+
+setup:: .transforms.setup
+
+# TRANSFORM_PYTHON_VERSION has no effect since requirements do not specify a python transform implementation
+set-versions: 
+	$(MAKE) TRANSFORM_PYTHON_VERSION=dummy TOML_VERSION=$(REPO_LVL_ORDER_RAY_VERSION) .transforms.set-versions 
+        
+build-dist:: set-versions .defaults.build-dist 
```

Comment:
you can/should remove set-versions here.

Reply:
fixed.